test/utils/UintUtils.ts

```diff
@@ -28,5 +28,78 @@ describe('UintUtils', function () {
         ).to.equal(ethers.constants.MaxUint256.toString());
       });
     });
+
+    describe('#toHexString', function () {
+      describe('(uint256)', function () {
+        it('returns 0 if input is 0', async () => {
+          const zeroNumber = ethers.BigNumber.from('0');
+          expect(
+            await instance.callStatic['toHexString(uint256)'](zeroNumber),
+          ).to.equal('0x00');
+        });
+
+        it('returns correct hexadecimal string representation of a number', async () => {
+          const inputValues = [
+            '1000',
+            '1',
+            '12345',
+            '85746201361230',
+            '999983',
+          ];
+          const outputValues = [
+            '0x03e8',
+            '0x01',
+            '0x3039',
+            '0x4dfc57df7b4e',
+            '0x0f422f',
+          ];
+          for (let i = 0; i < inputValues.length; i++) {
+            expect(
+              await instance.callStatic['toHexString(uint256)'](
+                ethers.BigNumber.from(inputValues[i]),
+              ),
+            ).to.equal(outputValues[i]);
+          }
+        });
+      });
+
+      describe('(uint256,uint256)', function () {
+        it('fails if length input is 0 and value is nonzero', async () => {
+          await expect(
+            instance.callStatic['toHexString(uint256,uint256)'](
+              ethers.BigNumber.from('100'),
+              ethers.BigNumber.from('0'),
+            ),
+          ).to.be.revertedWith('Strings: hex length insufficient');
+        });
```

Comment:
Put revert tests in a `describe('reverts if', ...)` block at the end of the parent describe block.